hack/build.sh

```diff
@@ -50,7 +50,11 @@ function squash {
 dirs=${VERSION:-$VERSIONS}
 
 for dir in ${dirs}; do
-  IMAGE_NAME="${BASE_IMAGE_NAME}-${dir//./}-${OS}"
+  if [ "$dir" == "2.6" ]; then
+    IMAGE_NAME=rhscl/rh-mongodb${dir//./}-${OS}
```

Comment:
OK. It could be this:
```
if [ "$dir" == "2.6" ]; then
    IMAGE_NAME=rhscl/rh-${BASE_DIR_NAME}${dir//./}-${OS}
else
    IMAGE_NAME="${BASE_IMAGE_NAME}-${dir//./}-${OS}"
fi
```
But `if` still have to be there.
Any idea?